docs/src/theme.js

```diff
@@ -179,6 +179,52 @@ export default {
       },
     },
   },
+  table: {
+    primary: {
+      container: {
+        overflowX: 'auto',
+        width: 750,
+        bg: 'white',
+      },
+      defaultTable: {
```

Comment:
I think you don't need the defaultTable theme here, you 
can provide in the component itself